Application/routing/router.go

```diff
@@ -9,7 +9,7 @@ func GenRouter() *gin.Engine {
 	router := gin.Default()
 	v1 := router.Group("/v1")
 	{
-		v1.GET("/hello_world/:id", actions.HelloWorldHandler)
+		v1.GET("/hello_world/:id", actions.GetInerest)
```

Comment:
точно hello world должен быть?

Reply:
не суть. это заглушка. там @ilyamazein сломал билд хитрыми манипуляциями с гитом, пришлось сделать так чтоб оно билдилось и трэвис не ругался

Reply:
@varlogen тут про path, возможно должно быть так
```suggestion
		v1.GET("/", actions.GetInerest)
```

Reply:
действительно, спасибо